static/js/components/SiteComponent.test.tsx

```diff
@@ -0,0 +1,52 @@
+const mockUseRouteMatch = jest.fn()
+
+import { fromPairs } from "lodash"
+
+import SiteComponent from "./SiteComponent"
+
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { makeWebsites } from "../util/factories/websites"
+import { siteComponentUrl } from "../lib/urls"
+
+import { Website } from "../types/websites"
+
+jest.mock("react-router-dom", () => ({
+  // @ts-ignore
+  ...jest.requireActual("react-router-dom"),
+  useRouteMatch: mockUseRouteMatch
+}))
+
+describe("SiteComponent", () => {
+  let helper: IntegrationTestHelper,
+    render: ReturnType<IntegrationTestHelper["configureRenderer"]>,
```

Comment:
we could add a type alias `TestRenderer` in `IntegrationTestHelper` to hold this type